.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
pls change the .json to yinka

Reply:
@ojoflorence please can you approve, what he did is actually right

Reply:
Yinka, please help us